apps_download/__init__.py

```diff
@@ -0,0 +1 @@
+import models
```

Comment:
Make this file more like https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py , add the header